pkgs/applications/emulators/wine/sources.nix

```diff
@@ -12,6 +12,15 @@ let fetchurl = args@{url, sha256, ...}:
   pkgs.fetchFromGitHub { inherit owner repo rev sha256; } // args;
     fetchFromGitLab = args@{domain, owner, repo, rev, sha256, ...}:
   pkgs.fetchFromGitLab { inherit domain owner repo rev sha256; } // args;
+
+    updateScriptPreamble = ''
+      set -eou pipefail
+      PATH=${with pkgs; lib.makeBinPath [ common-updater-scripts jq ]}:$PATH
+      sources_file=${__curPos.file}
```

Comment:
Can't we take that relative to the current file?

Reply:
What do you mean by this?

If by ‘current file’ you mean sources.nix, that's what `__curPos.file` evaluates to; taking that relative to itself would result in no information.

If by ‘current file’ you mean one of the update scripts, the update scripts are executed from the store. Their paths have nothing in common with the path of sources.nix, the writable file that they are meant to update.

Reply:
`${./sources.nix}` would be an eval-time construct, but not sure if it is any better…

Reply:
Wouldn't that import sources.nix into the store and reference that copy?